tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/SeleniumTests.cs

```diff
@@ -0,0 +1,163 @@
+                    return;
+                }
+
+                if (e.Value.PathAndQuery.EndsWith("api/v2/ci/tests/skippable"))
+                {
+                    e.Value.Response = new MockTracerResponse($"{{\"data\":[],\"meta\":{{\"correlation_id\":\"{correlationId}\"}}}}");
+                    return;
+                }
+
+                if (e.Value.PathAndQuery.EndsWith("api/v2/citestcycle"))
+                {
+                    e.Value.Headers["Content-Encoding"].Should().Be("gzip");
+
+                    var payload = JsonConvert.DeserializeObject<MockCIVisibilityProtocol>(e.Value.BodyInJson);
+                    if (payload.Events?.Length > 0)
+                    {
+                        foreach (var @event in payload.Events)
+                        {
+                            if (@event.Content.ToString() is { } eventContent)
+                            {
+                                if (@event.Type == SpanTypes.Test)
+                                {
+                                    tests.Add(JsonConvert.DeserializeObject<MockCIVisibilityTest>(eventContent));
+                                }
+                                else if (@event.Type == SpanTypes.TestSuite)
+                                {
+                                    testSuites.Add(JsonConvert.DeserializeObject<MockCIVisibilityTestSuite>(eventContent));
+                                }
+                                else if (@event.Type == SpanTypes.TestModule)
+                                {
+                                    testModules.Add(JsonConvert.DeserializeObject<MockCIVisibilityTestModule>(eventContent));
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            SetEnvironmentVariable("SAMPLES_SELENIUM_TEST_URL", $"http://127.0.0.1:{agent.Port}/evp_proxy/v4/rumpage");
+            using var processResult = await RunDotnetTestSampleAndWaitForExit(agent, packageVersion: packageVersion);
+
+            // Check if we have the data
+            tests.Count.Should().Be(1);
+            testSuites.Count.Should().Be(1);
+            testModules.Count.Should().Be(1);
+
+            var settings = VerifyHelper.GetCIVisibilitySpanVerifierSettings();
+            settings.UseTextForParameters("all_versions");
+            settings.DisableRequireUniquePrefix();
+            await Verifier.Verify(
+                tests
+                   .OrderBy(s => s.Resource)
+                   .ThenBy(s => s.Meta.GetValueOrDefault(TestTags.Parameters)),
+                settings);
+        }
+        catch
+        {
+            WriteSpans(tests);
+            throw;
+        }
```

Comment:
Meh, why not just rely on the snapshots which already do this?

Reply:
true!, changed in https://github.com/DataDog/dd-go/pull/132491/files